debian/snapd.postrm

```diff
@@ -53,6 +53,13 @@ if [ "$1" = "purge" ]; then
         rm -f "/etc/systemd/system/multi-user.target.wants/$unit"
     done
 
+    echo "Discarding preserved snap namespaces"
+    # opportunistic as those might not be actually mounted
+    for mnt in /run/snapd/ns/*.mnt; do
+        umount "$mnt" || true
+    done
+    umount /run/snapd/ns/ || true
```

Comment:
This is a bind mount "over itself" perhaps we could actually leave it alone entirely. I'll use --lazy "just in case"